src/repository.ts

```diff
@@ -113,6 +113,10 @@ abstract class Repository extends AsyncOptionalCreatable {
     this.execCommand('yarn build', silent);
   }
 
+  public test(silent = false): void {
```

Comment:
When would someone want this silent? WOuldn't you always want to see the test output? Is it for unit test?